pylint_odoo/test/main.py

```diff
@@ -8,7 +8,7 @@
 from pylint_odoo import misc
 
 
-EXPECTED_ERRORS = 52
+EXPECTED_ERRORS = 54
```

Comment:
As commented other times, it would be better to add a line like this:
```
EXPECTED_ERRORS += 2  # Errors due to C8106: Wrong version format
```
This way, multiple PRs don't get affected each other and the number seems less random.